server/chat-commands.js

```diff
@@ -3689,25 +3684,20 @@ const commands = {
 		}
 		if (!this.can('exportinputlog', null, room)) return;
```

Comment:
...Why? Why can't players export their own input log? It shouldn't be anyone's business if Player X and Player Y both consent to extracting their own input logs.

Reply:
They can...

Reply:
No? Only Leaders have `exportinputlog` set to true, Administrators have `exportinputlog` set to true by default, because they have root access,

Reply:
Try it on Main. I already changed it, I just haven't updated the example config yet.

Reply:
Oh, that makes sense. Created #5457 to fix that.